client/app/dbaas/logs/detail/streams/home/logs-streams-home.controller.js

```diff
@@ -1,7 +1,88 @@
 class LogsStreamsHomeCtrl {
-    constructor ($stateParams) {
+    constructor ($q, $state, $stateParams, LogsStreamsService) {
+        this.$q = $q;
+        this.$state = $state;
         this.$stateParams = $stateParams;
         this.serviceName = this.$stateParams.serviceName;
+        this.LogsStreamsService = LogsStreamsService;
+    }
+
+    $onInit () {
+        this.init();
+    }
+
+    /**
+     * initializes streams and quota object by making API call to get data
+     *
+     * @memberof LogsStreamsHomeCtrl
+     */
+    init () {
+        this.loading = true;
+        // get total and used streams
+        const quota = this.LogsStreamsService.getQuota(this.serviceName)
+            .then(results => {
+                this.quota = results;
+            });
+        // get streams
+        const streams = this.LogsStreamsService.getStreams(this.serviceName)
+            .then(results => {
+                this.streams = results;
+            });
+        // show loading spinner untill all streams are loaded
+        this.$q.all([quota, streams])
+            .finally(() => {
+                this.loading = false;
+            });
+    }
+
+    /**
+     * takes to options page
+     *
+     * @memberof LogsStreamsHomeCtrl
+     */
+    increaseQuota () {
+        this.$state.go("dbaas.logs.detail.options", {
```

Comment:
Remove and use ui-sref instead if you can

Reply:
used ui-sref